### PR TITLE
refactor integration test setup

### LINKED ISSUE #360: Improve integration test automation
Updating the integration tests to use the new gaia binary has involved manually going to the gaia repo, using the make commands there to push to your personal docker repository, replacing the image and tag in this repo in `test_chains.go` and then running the `make test-gaia`, if the test fails, you need to find the logs for the container used and then debug the issue. This is documented in #357 

This process should become automated and the scripts which might break should be located in this repo so we have more control and visibility for when they do break. Copy/pasting the Dockerfile.test from gaia and the single node script is a start. Ideally we should be able to use a standard gaia docker image rather than creating our own

### PR DESCRIPTION
**Pull request by colin-axner**
Refactor integrations tests to build a docker image with the specified configurations locally rather than relying on a remote docker image

closes: #360

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
 COMMIT  := $(shell git log -1 --format='%H')
 SDKCOMMIT := $(shell go list -m -u -f '{{.Version}}' github.com/cosmos/cosmos-sdk)
-GAIA_VERSION := v4.0.0
+GAIA_VERSION := v4.1.0
 AKASH_VERSION := jack/update-sdk
```
Comment (fedekunze) [inline Makefile]:
```suggestion
AKASH_VERSION := v0.10.2
```

File: test/setup/Dockerfile.akashtest [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# https://hub.docker.com/r/ovrclk/akash/tags?page=1&ordering=last_updated
+FROM ovrclk/akash:0.10.0
```
Comment (fedekunze) [inline test/setup/Dockerfile.akashtest:2]:
```suggestion
FROM ovrclk/akash:0.10.2
```

**Review by fedekunze — APPROVED** [review]
LGTM, what are the instructions to run this locally?

File: test/setup/Dockerfile.akashtest (hunk 2 above)
Comment (colin-axner) [reply test/setup/Dockerfile.akashtest:2]: I don't think they have a docker image for that version

**Comment by colin-axner** [comment]
> LGTM, what are the instructions to run this locally?

`make test-gaia`
`make test-akash`

**Pull request merged**